clang/include/clang/Sema/Sema.h

```diff
@@ -12998,6 +12998,11 @@ class Sema final {
   void MarkDevice();
   void MarkSyclSimd();
 
+  /// Emit a diagnostic about the given attribute having a deprecated name, and
+  // also emit a fixit hint to generate the new attribute name.
```

Comment:
```suggestion
  /// also emit a fixit hint to generate the new attribute name.
```
Should this line be a Doxygen comment as well?

Reply:
Good catch! I've corrected it.